### DISCUSSION TITLE
Pressure loss and aerodynamic free area

### DISCUSSION THREAD

**Discussion by ShehabGamrah** [firemodels/fds#13946] [Q&A]
![{765DC659-D12F-49BC-B4A6-E69992A7C663}](https://github.com/user-attachments/assets/9c0bdd11-c3e8-4ab0-bc01-0ae44a98391d)
![{0A533A21-1AC9-4B3B-AC4E-2DAA574079E5}](https://github.com/user-attachments/assets/c636dede-12ca-4531-9780-4463143064d1)
I have specified a duct with properties to represent flow restriction due to aerodynamic free area. However as seen in the results the pressure near the inlet drops below 0pa to -8pa. Is there a way to resolve this so pressure stays 0 Pa near the inlet. As this effects the pressure within the stair.

**Comment by Er9y714** [comment]
Pressure difference is an outcome, not a parameter. Due to resistance of the opening, the pressure difference occurs. Do you think the results are not realistic?

**Reply by ShehabGamrah** [reply]
Yes, I understand. My issue is that the pressure drops below 0 Pa even before it enters the orifice. It reaches -8 Pa. Shouldn’t the pressure remain at 0 Pa before the orifice?

**Reply by drjfloyd** [reply]
FDS decomposes pressure into the background pressure (which inlcudes hydrostatic) and the perturbation pressure. The pressure that HVAC uses is the total pressure which is the background plus the perturbation.  Your output is just the perturbation pressure. Also FDS HVAC uses the average condition over the entire HVAC VENT which your output is not showing (just a slice through the VENT). Look at the total pressure or just output the HVAC node pressure

**Reply by ShehabGamrah** [reply]
![image](https://github.com/user-attachments/assets/d97a74c6-4c63-43a5-b53c-47f465549d94)

What I mean is the pressure before the air even enters the orifice marked in red rectangular drops below atmospheric pressure. However, the pressure should remain constant at 0 Pa before entering the orifice, and the loss should occur through the orifice. When I specify the first node as an ambient node, this problem is resolved. However, I would like to know how to fix this issue when the first node is not set to ambient and is connected to a wider atmosphere. Alternatively, could this drop occur before the flow enters the orifice be the correct phenomenon? If the first node is not set to ambient and connects to a wider atmosphere, or could this drop occur before the flow enters the orifice be the correct phenomenon?

**Comment by drjfloyd** [comment]
You need to look at total pressure and not just the perturbation pressure.
Look at the node pressure outputs for the HVAC nodes.

**Reply by ShehabGamrah** [reply]
I am not speaking about pressure at nodes. The pressure before flow enter the nodes drops below the atmospheric pressure.

**Reply by drjfloyd** [reply]
What specific QUANTITY are you showing in your slice file?  Using the Technical Reference Manual, what variable does that QUANTITY plot?  Is it the same as what determine flow for the HVAC model?

**Reply by ShehabGamrah** [reply]
![image](https://github.com/user-attachments/assets/af92251b-a4c8-4fd0-ae01-c1e07530a505)
The is output pressure. As you can see, this is my orifice, and the flow before it even enters the orifice drops. Is this realistic? I expected the flow to be ambient and drop through the orrifice.

**Reply by drjfloyd** [reply]
Attach a SIMPLE version of your inputs that demonstrates what you are seeing.

**Reply by ShehabGamrah** [reply]
[extendedboundary.txt](https://github.com/user-attachments/files/18253549/extendedboundary.txt)
[ambientnode.txt](https://github.com/user-attachments/files/18253558/ambientnode.txt)

Attached are two case files: one where the inlet node is not set to ambient and another where it is forced to ambient. The issue has two aspects. First, the pressure drops below ambient even before entering the HVAC duct, but this is resolved when the inlet node is set to ambient. Second, I am attempting to model a discharge coefficient of 0.7. Using the formula LOSS_COEFFICIENT_F = 1 / Cd², I set both the forward and reverse loss coefficients to 2.04 to achieve the desired effect. However, the pressure drop through the duct is slightly lower than expected. Could this indicate an issue with my relationship between the forward loss coefficient and the discharge coefficient?
​

**Reply by drjfloyd** [reply]
I had asked what QUANTITY are you plotting and what pressure term that is.  The answers are you are plotting PRESSURE which is the perturbation pressure or p tilde in Eq. 3.1 in the Tech Ref. HVAC uses the total pressure or p on the left side of Eq 3.1.

You have this input:

&SURF ID='Extraction01',
      RGB=26,128,26,
      VOLUME_FLOW=5.0/

At t=0 this is going to start removing air from inside your domain.  What is this going to do? It is going to drop the background pressure inside (the p bar m term in Eq 3.1). What is now happening at the top of the stair?  The background pressure inside the building is dropping below ambient which means the total pressure is droppping. Outside the building, nothing has changed. You now have lower pressure inside the hatch than outside.  Air will start to flow through the HVAC vent into the building. What has to happen in the FDS gas phase domain outside the building? For air to flow in the FDS gas phase domain there must be a gradient in the perturbation pressure. So outside the hatch, for air to flow down into the hatch there must be a gradient in the perturbation pressure for air to flow along. Hence you see a perturbation pressure gradient at the hatch as air is flowing towards the hatch.  

Adding slices of ABSOLUTE PRESSURE and BACKGROUND PRESSURE might help you better understand what is happening.

A 5 m3/s flow in a 1 m2 duct will be 5 m/s.  Quick hand calc (dP = 1/2 rho U^2 K)  assuming a density of 1.2 kg/m3 for air gives 30.4 Pa of pressure drop. The DEVC for the duct gives 28.5 Pa. This is just the change in pressure. We also have the weight of the column of air in the duct (the rho g z term in Eq 11.3) which adds another 2 Pa.  Look at the duct node pressure difference output. At t=0 when there is no flow, the pressure difference is not zero it is +2 Pa which is just the hydrostatic pressure gradient or the weight of the air in the duct.

**Reply by ShehabGamrah** [reply]
![{B057E773-AD61-439E-9153-226ECF1FB16E}](https://github.com/user-attachments/assets/9e5a02e0-bcef-4efc-9d26-70da6d6ada0f)

![{B52680BB-51B9-42CC-9E61-044157715750}](https://github.com/user-attachments/assets/d76af2c6-97b1-4b92-b8bd-f11ff20a0613)

![{17D61532-A9D7-4B9C-B2C5-58685092FDB1}](https://github.com/user-attachments/assets/0222cbc5-d565-49db-ab3b-2b87b3e2ebc7)


I have added slice of background and total pressure and it seems confusing that air at the bottom inside the building is higher than outside. Shouldn't air flow from higher to lower pressure, and this is what is shown using the normal pressure output. So what is the significance of the total and background pressure slices as they are opposite of the perturbation pressure.

**Reply by drjfloyd** [reply]
You need to be thinking in terms of the net forces on a parcel of fluid.

If you have an open top bucket of water is the pressure at the bottom of the bucket higher or lower than the pressure at the top?
It is higher. Why? Because compared to the top of the bucket where you just have the weight of the air you also have the added weight of the water. Same thing happens in a shaft of air. Pressure at the bottom is higher than the top because there is more weight of air.  Does the water at the bottom of a bucket spontaneously flow up and out of the bucket? The pressure is higher at the bottom, why doesn't the bucket spontaneously empty itself of water? Because the increase in pressure at the bottom is exactly equal to the weight of the water. The upward force of the pressure is equal to and opposite the downward force of the weight. The two cancel out and zero net force means zero motion.

The same is true in an air filled shaft. Pressure at the bottom is higher than pressure at the top. If all you have is the hydrostatic pressure and gravity acting on the air, then the forces balance and there is no fluid movement. Now take that still shaft of air and apply a very tiny  amount of suction at the bottom of the shaft. The moment you drop the local pressure at the point of suction just a tiny amount below the hydrostatic pressure, the forces are now unbalanced. At the point of suction, the weight of fluid now exceeds the hydrostatic force. The pressure at the bottom is still higher than the pressure at the top but the net force on a parcel of fluid at the suction point is now unbalanced.

Within a pressure zone, perturbation pressure is what causes fluid to move. Between pressure zones (i.e inside and outside) when using HVAC you need to look at the total pressure for the HVAC nodes to see where fluid moves. So in your case above the hatch the perturbation pressure is negative. Which it must be in order for gas to flow into the hatch. The background pressure is the same. This means just outside the hatch the total pressure is reduced. However, inside the building the background pressure has dropped. It has dropped by a larger amount than the perturbation pressure drop outside the hatch. This means there is a duct pressure gradient from inside to outside and flow moves into the building.

**Reply by ShehabGamrah** [reply]
Thanks for your time and explanation. However, to close this off, I have a question about pressure. I need to check the deprressurization within the corridor at ground level due to extraction, but I don’t want the pressure difference between the corridor and other areas to exceed a certain value. So, for this value, I need to use perturbation pressure, and the hydrostatic pressure within the stair has no significance.

Is the only significance of hydrostatic pressure within the HVAC orifice pressure loss? Therefore, I want to ask if I should calculate the pressure drop before air enters the orifice all the way to the bottom level. I would calculate the pressure drop through the orifice and the pressure drop per each stair level, and that would be the perturbation pressure. So, how can I include the HVAC +2 Pa in this slice?

Secondly, in analytical examples, people calculate the pressure drop within an orifice by assuming that the pressure entering the orifice is 0 Pa. However, in my case, the flow entering the orifice is -8 Pa. So, is the -8 Pa the correct value, or should the flow stay at 0 Pa entering the orifice?

**Comment by drjfloyd** [comment]
The orifice flow formula just gives the pressure drop through the orifice. It does not require that the pressure at the orifice be zero.  

Within a pressure zone, the only thing that drives flow in FDS is the perturbation pressure. If you have multiple pressure zones (in your case inside and outside), then flow between pressure zones through HVAC systems uses the absolute pressure.  What you want is the difference in absolute pressure between the inside of the building and the outside.

**Reply by ShehabGamrah** [reply]
![](https://a.storyblok.com/f/79904/2754x1501/18d2e981b6/de-pressurisation-01.png)

![image](https://github.com/user-attachments/assets/78b8ea02-157d-4971-9e97-771da4cff60e)

I have uploaded images of what I am modelling at ground level and how air flows into the stair vertically. What I am trying to calculate is the pressure within the corridor as the air pressure drops from the inlet before it enters the vent at the head of the stair all the way through the corridor. And there are formulas for pressure drop per stair level. What is left is the pressure drop as air enters the orrifice ( top vent). How can I output this pressure in FDS, if the perturbation pressure doesn’t include the full pressure drop of the orrifice. Yet the perturbation pressure is the essential pressure rather than outputting total pressure. Secondly, is perturbation pressure expected to drop -8 pa before flow enters the top vent.

**Reply by drjfloyd** [reply]
I can't do your engineering analysis for you. The User's Guide gives all the possible pressure outputs.
